src/NUnitFramework/framework/Constraints/MsgUtils.cs

```diff
@@ -536,99 +555,169 @@ public static int[] GetArrayIndicesFromCollectionIndex(IEnumerable collection, l
         /// string with ellipses representing the removed parts
         /// </summary>
         /// <param name="s">The string to be clipped</param>
-        /// <param name="maxStringLength">The maximum permitted length of the result string</param>
+        /// <param name="clipLength">The length of the clipped string</param>
         /// <param name="clipStart">The point at which to start clipping</param>
         /// <returns>The clipped string</returns>
-        public static string ClipString(string s, int maxStringLength, int clipStart)
+        public static string ClipString(string s, int clipLength, int clipStart)
         {
-            int clipLength = maxStringLength;
             StringBuilder sb = new StringBuilder();
```

Comment:
Should we set this to be of a certain size - e.g. `s.length` + 6?

Reply:
In case anyone ever changes the value of `ELLIPSIS`, I changed it to `s.Length + 2 * ELLIPSIS.Length`